substrate/frame/babe/Cargo.toml

```diff
@@ -39,53 +34,41 @@ pallet-balances = { workspace = true, default-features = true }
 pallet-offences = { workspace = true, default-features = true }
 pallet-staking = { workspace = true, default-features = true }
 pallet-staking-reward-curve = { workspace = true, default-features = true }
-sp-core = { workspace = true, default-features = true }
 
 [features]
 default = ["std"]
 std = [
-	"codec/std",
-	"frame-benchmarking?/std",
-	"frame-election-provider-support/std",
-	"frame-support/std",
-	"frame-system/std",
-	"log/std",
-	"pallet-authorship/std",
-	"pallet-balances/std",
-	"pallet-offences/std",
-	"pallet-session/std",
-	"pallet-staking/std",
-	"pallet-timestamp/std",
-	"scale-info/std",
-	"sp-application-crypto/std",
-	"sp-consensus-babe/std",
-	"sp-core/std",
-	"sp-io/std",
-	"sp-runtime/std",
-	"sp-session/std",
-	"sp-staking/std",
+    "codec/std",
+    "frame/std",
+    "log/std",
+    "pallet-authorship/std",
+    "pallet-balances/std",
```

Comment:
Should not be needed if these pallets are only needed for tests.

Reply:
Will test it out after I have something stable, I left dependency features as they were only replacing the ones that were affecting by frame crate.